src/main/services/sealedClaimPdfGenerator.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express'
+
+import { Claim } from 'claims/models/claim'
+import { DocumentsClient } from 'documents/documentsClient'
+import { pdfEndpointResponseHandler } from 'services/pdfEndpointsResponseHandler'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+export class SealedClaimPdfGenerator {
+
+  /**
+   * Handles sealed claim downloads
+   *
+   * Note: Middleware expects to have {@link Claim} available in {@link express.Response.locals}
+   *
+   * @param {e.Request} req HTTP request
+   * @param {e.Response} res HTTP response
+   * @param {e.NextFunction} next next handler
+   * @returns {Promise<void>}
+   */
+  static async requestHandler (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> {
+    const claim: Claim = res.locals.claim
+
+    documentsClient.getSealedClaimPDF(claim.externalId, res.locals.user.bearerToken)
+      .on('response', pdfEndpointResponseHandler(`sealed-claim-${claim.claimNumber}`, res, next))
```

Comment:
is this the correct file name?

Reply:
Isn't it? I'm happy to change it if there is a better name

Reply:
<img width="710" alt="screen shot 2018-05-04 at 02 12 23" src="https://user-images.githubusercontent.com/1724691/39609410-a4d2819c-4f40-11e8-9a3c-76d74b89510d.png">

I guess Tim though to have consistency. See screenshot attached. Please make file name start with case reference number.